client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -70,35 +69,41 @@ export default React.createClass( {
 				showStore: true
 			} );
 
-			if ( this._pressableStore ) {
-				this._pressableStore.focus();
+			if ( this.pressableStore ) {
+				this.pressableStore.focus();
 			}
 
 			return;
 		}
 
 		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], { designType } );
 		this.props.goToNextStep();
-	},
+	};
 
-	renderChoice( choice ) {
+	renderChoice = ( choice ) => {
 		return (
 			<Card className="design-type-with-store__choice" key={ choice.type }>
-				<a className="design-type-with-store__choice__link" href="#" onClick={ ( event ) => this.handleChoiceClick( event, choice.type ) }>
+				<a className="design-type-with-store__choice__link"
+					href="#"
+					onClick={ ( event ) => this.handleChoiceClick( event, choice.type ) }>
```

Comment:
the linter will complain about passing a fat-arrow function inside of an event callback. you will want to either move this out or add some linting flag.

it doesn't appear like you can eliminate the need of creating a closure here, so the linting rule is somewhat moot, however, you can make it happy _and_ clean up the way it looks a bit (in my opinion)

```js
handleChoiceClick = type => event => {
	event.preventDefault();
	event.stopPropagation();
	this.handleNextStep( type );
}

onClick={ this.handleChoiceClick( choice.type ) }
```

In this case, we're still creating a new function on every call, but the linter won't complain and we don't have to name and pass around that `event` object